mslib/utils/qt.py

```diff
@@ -385,9 +384,6 @@ def __init__(self, function):
             self.finished = NonQtCallback()
             self.failed = NonQtCallback()
 
-        self.failed.connect(lambda e: self._update_gui())
-        self.finished.connect(lambda x: self._update_gui())
```

Comment:
What's the reasoning for removing these? I wouldn't be surprised if this breaks some of the other uses of the Worker class, but I also have no idea why this was here anyway.